I can't reproduce this test failure locally and am not quite sure what would be different on Travis:

```
sympy/physics/mechanics/tests/test_linearize.py:test_linearize_rolling_disc_kane 

Traceback (most recent call last):

  File "sympy/utilities/runtests.py", line 1317, in _timeout

    function()

  File "sympy/utilities/pytest.py", line 155, in func_wrapper

    func()

  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/sympy-1.6.dev0-py2.7.egg/sympy/physics/mechanics/tests/test_linearize.py", line 84, in test_linearize_rolling_disc_kane

    assert linearizer.f_a == f_v.diff(t)

AssertionError

 tests finished: 54 passed, 1 failed, 5 skipped, 9 expected to fail, 

in 1272.57 seconds 

DO *NOT* COMMIT!

Traceback (most recent call last):

  File "<stdin>", line 4, in <module>

Exception: Tests failed
```

--- a/sympy/physics/mechanics/tests/test_linearize.py
+++ b/sympy/physics/mechanics/tests/test_linearize.py
@@ -81,7 +81,7 @@ def test_linearize_rolling_disc_kane():
     linearizer = KM.to_linearizer()
     assert linearizer.f_c == f_c
     assert linearizer.f_v == f_v
-    assert linearizer.f_a == f_v.diff(t)
+    assert linearizer.f_a == f_v.diff(t).subs(KM.kindiffdict())
     sol = solve(linearizer.f_0 + linearizer.f_1, qd)
     for qi in qdots.keys():
         assert sol[qi] == qdots[qi]

